src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -1505,6 +1505,23 @@ def test_vmss_existing_lb(self, resource_group):
         self.cmd('network lb create -g {rg} -n {lb} --backend-pool-name test')
         self.cmd('vmss create -g {rg} -n {vmss} --load-balancer {lb} --image UbuntuLTS --admin-username clitester --admin-password TestTest12#$')
 
+    @ResourceGroupPreparer()
+    def test_vmss_single_placement_group_default_to_std_lb(self, resource_group):
+        self.kwargs.update({
+            'vmss': 'vmss123'
+        })
+        self.cmd('vmss create -g {rg} -n {vmss} --admin-username clitester --admin-password PasswordPassword1! --image debian --single-placement-group false')
```

Comment:
The logic is getting very complex for scale sets. I'd recommend putting a comment here on what the expected behavior is and why,